app/V1Module/presenters/GroupsPresenter.php

```diff
@@ -204,6 +224,60 @@ public function actionUpdateGroup(string $id) {
     $this->sendSuccessResponse($this->groupViewFactory->getGroup($group));
   }
 
+  /**
+   * Set the 'isOrganizational' flag for a group
+   * @POST
+   * @Param(type="post", name="value", validation="bool", required=TRUE, description="The value of the flag")
+   * @param string $id An identifier of the updated group
+   * @throws ForbiddenRequestException
+   * @throws InvalidArgumentException
+   * @throws NotFoundException
+   */
+  public function actionSetOrganizational(string $id) {
```

Comment:
Not that I don't appreciate specialized endpoints, but I thought this will be set in the group edit form.